libraries/vidazooUtils/bidderUtils.js

```diff
@@ -500,3 +505,30 @@ export function createBuildRequestsFn(createRequestDomain, createUniqueRequestDa
     return requests;
   }
 }
+
+export function getAndSetFirstPartyData(storageRef) {
```

Comment:
This appears to be a rules violation, what are you trying to do. Your pr name appears to be an attempt at deception